.github/CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,3 @@
+# Code of Conduct
+
+Refer to our [Code of Conduct](https://github.com/tinkerbell/.github/blob/master/CODE_OF_CONDUCT.md)
```

Comment:
Github will look up the CoC from tinkerbell/.github automagically so this file is unnecessary.

Reply:
According to https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/creating-a-default-community-health-file